### DISCUSSION TITLE
Non-environment variables in Kubernetes deployment

### DISCUSSION THREAD

**Discussion by columbian-ath** [DefectDojo/django-DefectDojo#6942] [Q&A]
Is it possible to configure DEDUPLICATION_ALGORITHM_PER_PARSER and HASHCODE_FIELDS_PER_SCANNER parameters if Defectdojo is deployed in the Kubernetes cluster? According to https://documentation.defectdojo.com/getting_started/configuration/ only environment variables can be modified. I will be thankful for every piece of advise!

**Comment by mykhailo-sindieiev** [comment]
Hello, if you define these variables as DD_HASHCODE_FIELDS_PER_SCANNER and DD_DEDUPLICATION_ALGORITHM_PER_PARSER so they will be used by the code [here](https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/settings/settings.dist.py#L1256) and [here](https://github.com/DefectDojo/django-DefectDojo/blob/master/dojo/settings/settings.dist.py#L1449) instead of default
